fmf/base.py

```diff
@@ -558,6 +562,9 @@ def prune(self, whole=False, keys=None, names=None, filters=None,
             if names and not any(
                     [re.search(name, node.name) for name in names]):
                 continue
+            # Select nodes defined by any of the source files
+            if sources and not bool(sources.intersection(node.sources)):
```

Comment:
```suggestion
            if sources and not sources.intersection(node.sources):
```

Or is there a specific reason why explicitly convert to `bool` here?

Reply:
Frankly I have not idea.

Reply:
Ok, removed then.